README.md

```diff
@@ -5,6 +5,21 @@
 Holds generated but persistent results from building the `ansible` community package.  This information
 may be referred to by other projects and scripts.
 
+## Issue tracker
+
+The [issue track of this repository](https://github.com/ansible-community/ansible-build-data/issues) is to track various aspects of the `ansible` build, including:
+
+1. Tracking release dates,
+1. Tracking blockers for a release,
+1. Tracking adding, renaming, and removing collections,
+1. Tracking problems with a release related to the build process:
+   - This includes problems that prevent the package to be installed or system packages to be built from the PyPI release;
+1. Tracking and discussing other problems with the `ansible` community package:
+   - This includes important problems with the included collections that are not reacted on by the collection maintainers, for example largescale incompatibilities with the current ansible-core version, violation of semantic versioning, and general violation of the [Ansible inclusion requirements](https://docs.ansible.com/ansible/devel/community/collection_contributors/collection_requirements.html);
+   - This includes major or security bugs in collections with wide-reaching consequences that are not addressed by the collection maintainers, or cannot be addressed on the collection level for some reason.
+
+This issue tracker is **not** for tracking regular bugs, feature requests, or for asking for help with collections included in the `ansible` package. **Such issues will be closed.** Instead, check out the issue trackers of the respective collections, or consider [asking for help in the Ansible forum](https://forum.ansible.com/).
```

Comment:
I think we should also mention ansible-core here. This is not for tracking issues with ansible-core and those issues should be reported to github.com/ansible/ansible.

Reply:
@gotmax23 In all those lines? I think it's a good idea to add ansible-core (in addition to included collections) to what issues here are **not** for, something like:

> This issue tracker is **not** for tracking regular bugs, feature requests, or for asking for help with collections included in the `ansible` package or `ansible-core`. **Such issues will be closed.** Instead, check out the issue trackers of the respective collections or [ansible-core](https://github.com/ansible/ansible), respectively, or consider [asking for help in the Ansible forum](https://forum.ansible.com/).

But I don't see ansible-core in the other lines you've mentioned. I may be wrong, though. It might help if you would provide a suggestion.

Reply:
I would split the sentence up, like this:
```suggestion
This issue tracker is **not** for tracking regular bugs, feature requests, or for asking for help with collections included in the `ansible` package. It is also not for issues with `ansible-core`. **Such issues will be closed.** Instead, check out the issue trackers of the respective collections, the [ansible-core issue tracker](https://github.com/ansible/ansible/issues/), or consider [asking for help in the Ansible forum](https://forum.ansible.com/).
```

Reply:
Sounds good to me. @gotmax23?

Reply:
@mariolenz, I was only talking about the last paragraph. I'm not sure why Github showed all that context.

How about this?

```suggestion
This issue tracker is **not** for tracking regular bugs or feature requests for `ansible-core` or the collections in included in the `ansible` package or for user support. **Such issues will be closed.** Instead, check out the [`ansible-core` issue tracker](https://github.com/ansible/ansible), issue trackers of the respective collections, or consider [asking for help in the Ansible forum](https://forum.ansible.com/).
```